src/components/ec-inline-actions/ec-inline-actions.vue

```diff
@@ -74,61 +77,78 @@ export default {
       default: () => ({ offset: 10, placement: 'bottom-start' }),
     },
   },
+  methods: {
+    componentTag(item) {
+      if (item.href) {
+        return 'a';
+      }
+      return 'button';
+    },
+    getDownloadText(item) {
+      const hasHrefAndDowload = !!item.href && !!item.download;
+      if (hasHrefAndDowload) {
+        return item.download;
+      }
+      if (item.href) {
+        return '';
+      }
+      return null;
+    },
+  },
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-
+<style>
 .ec-inline-actions {
-  margin: 0;
-  padding: 0;
+  @apply tw-m-0;
+  @apply tw-p-0;
+
   min-width: 160px;
   max-width: 320px;
 
   &__item {
-    list-style: none;
-    width: 100%;
+    @apply tw-list-none;
+    @apply tw-w-full;
   }
 
   &__button {
-    text-align: left;
-    width: 100%;
-    background: $white;
-    color: $level-3-body-and-headings;
-    border: none;
-    font-size: 16px;
-    cursor: pointer;
-    padding: 8px 16px;
-    display: flex;
-    align-items: center;
+    @apply tw-text-left;
+    @apply tw-w-full;
+    @apply tw-bg-gray-8;
+    @apply tw-text-gray-3;
+    @apply tw-border-none;
+    @apply tw-btn-text;
```

Comment:
:+1: Like it. Make sure the line-height: 22px and font-weight: 400 doesn't break the design.

Reply:
It doesn't break it!